### ISSUE TITLE
"Elevio not loaded yet..." warning using react component

### ISSUE THREAD

**Issue by cduff** [dixahq/elevio-client-packages#13]
I recently upgraded from react 17 to 18 and now I get the following warning in the console during development whenever I view my app:

```
Elevio not loaded yet, please wait until Elevio.load has completed.
```

I have created a CodeSandbox to reproduce:

https://codesandbox.io/s/elevio-not-loaded-yet-deltxk

Interestingly, the error can be avoided by removing `StrictMode` or by using the old react 17 render method (see `index.tsx`). Obviously, we don't want to be forced to do either of those things thanks to Elevio though.

Can this be fixed so we don't see warnings in our dev console thanks to Elevio?

**Comment by kristian-puccio** [comment]
Sorry to hear about the issues with React 18, I'll take a look at this for you soon.

Thanks for the Sandbox.

**Comment by kristian-puccio** [comment]
Fixed with version 1.3.4

**Comment by cduff** [comment]
Thanks @kristian-puccio. Confirmed fixed for me by 1.3.4.
